charts/images.yaml

```diff
@@ -1,16 +1,16 @@
 images:
 - name: terraformer
-  sourceRepository: github.com/gardener/terraformer
+  sourceRepository: https://github.com/gardener/terraformer
   repository: eu.gcr.io/gardener-project/gardener/terraformer
-  tag: "v1.5.0"
+  tag: v2.0.0
```

Comment:
We just released [Terraformer v2.1.0](https://github.com/gardener/terraformer/releases/tag/v2.1.0), so there is now a Packet-specific image that only contains the TF Packet provider plugin, hence, is much smaller! See also https://github.com/gardener/gardener-extension-provider-aws/pull/241

```yaml
- name: terraformer
  sourceRepository: https://github.com/gardener/terraformer
  repository: eu.gcr.io/gardener-project/gardener/terraformer-packet
  tag: v2.1.0
```

Reply:
Cool, will update and test.

Reply:
LGTM. I've pushed the update.